CondFormats/HcalObjects/src/HcalInterpolatedTableFunctor.cc

```diff
@@ -0,0 +1,101 @@
+#include <algorithm>
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include "CondFormats/HcalObjects/interface/HcalInterpolatedTableFunctor.h"
+
+inline static double interpolateStep(
```

Comment:
Hm... this behavior is against the C++ standard. Are you sure
that you are not using some special switch, like -fkeep-inline-functions
or -fkeep-inline-functions?

On 08/04/2016 05:42 PM, Slava Krutelyov wrote:
>
> In CondFormats/HcalObjects/src/HcalInterpolatedTableFunctor.cc <https://github.com/cms-sw/cmssw/pull/15324#discussion_r73615584>:
>
> > @@ -0,0 +1,101 @@
> > +#include <algorithm>
> > +#include "FWCore/Utilities/interface/Exception.h"
> > +
> > +#include "CondFormats/HcalObjects/interface/HcalInterpolatedTableFunctor.h"
> > +
> > +inline static double interpolateStep(
>
> recompiled with -O0:
>
> |nm ../lib/slc6_amd64_gcc530/libCondFormatsHcalObjects.so | grep interpolateStep | c++filt 000000000027aaea t interpolateStep(double, double, double, double, double) |
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/cms-sw/cmssw/pull/15324/files/f2734c7b29b226480f1efc887de88c8ae73f121f#r73615584>, or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFS4-bmxgN0GKFLVCv17pUZJvS9FWvHGks5qcmrEgaJpZM4JXio9>.
>